pkg/smartcontract/entry.go

```diff
@@ -20,6 +20,60 @@ import (
 // for interactions with RPC server that have iterator sessions disabled.
 func CreateCallAndUnwrapIteratorScript(contract util.Uint160, operation string, maxIteratorResultItems int, params ...any) ([]byte, error) {
 	script := io.NewBufBinWriter()
+	jmpIfNotOffset, jmpIfMaxReachedOffset := emitCallAndUnwrapIteratorScript(script, contract, operation, maxIteratorResultItems, params...)
```

Comment:
We have a test for it, but it's not directly checks this API, see the `TestClient_InvokeAndPackIteratorResults`. We can extend this test.

Reply:
OK, we currently don't have this test, it was renamed and currently it checks iterator sessions. See the `TestClient_IteratorSessions`. So could we extend this sessions-based test?

Reply:
It is somewhat different in spirit (tests iteration behaviour, not our script)
I have added `unwrap` tests, though.